README.md

```diff
@@ -3,11 +3,38 @@ A [Kanister](https://github.com/kanisterio/kanister) backup management system.
 
 Taweret sets retention periods for Kanister backups and deletes them once they expire by interacting with Kanister CRDs.
 
-This project is in a very early development phase. Please check the issues tracker for planned features, or to submit any feature requests.
+This project is in an early development phase. Please check the issues tracker for planned features, or to submit any feature requests.
 
 ## How to
 
-Taweret should be deployed to a Kubernetes cluster which already runs Kanister. A Taweret Helm chart does not yet exist.
+Taweret should be deployed to a Kubernetes cluster which already runs Kanister. 
+
+### Taweret
+
+Taweret can be installed through its Helm chart:
+
+    helm repo add renku https://swissdatasciencecenter.github.io/helm-charts
+    helm install taweret renku/taweret
+
+Backup configurations can be defined in the Helm values file. The default backup configuration is:
+
+    backupConfigs:
+      daily-postgres:
+        name: daily-postgres
+        kanisterNamespace: kanister
+        blueprintName: postgres-bp
+        profileName: default-profile
+        retention:
+          backups: 7
+          minutes: 0
+          hours: 0
+          days: 7
+          months: 0
+          years: 0
+
```

Comment:
For me, it was unclear where the postgres stuff comes from here; iiuc, in principle, tawaret and kanister can make backups for quite arbitrary applications - I guess the `blueprint` indicates where the postgres db lives - is that correct?

Reply:
The Postgres stuff in the documentation is just for demonstration purposes - although this is how we use it at SDSC. The `blueprint` contains the backup commands and their steps - i.e. backup, delete, restore. We require those so we know how actually to delete the backups. The actual specifying of the statefulset is done in the CronJob, entirely outside of Taweret. Taweret doesn't need to know where the Postgres DB lives because it doesn't take the backups - it just manages them.

Reply:
Exactly, yes